app/assets/javascripts/govuk_publishing_components/lib/govspeak/magna-charta.js

```diff
@@ -215,8 +215,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       return !isNaN(parseFloat(val))
     },
     stripValue: function (val) {
-      var re = new RegExp('\\,|£|%|[a-z]', 'gi')
-      return val.replace(re, '')
+      return val.replace(/,|£|%|[a-z]/gi, '')
```

Comment:
We're not using any dynamic input in this regex to require a constructor, so the conversion shouldn't bite us, but I do share the 'dubious' feeling.